share/src/main/resources/alfresco/slingshot-application-context.xml

```diff
@@ -60,6 +64,19 @@
          </list>
       </constructor-arg>
    </bean>
+    
+    <bean id="web.config"
+		class="org.springframework.extensions.config.xml.XMLConfigService"
+		init-method="init">
+		<constructor-arg>
+			<ref bean="webframework.configsource" />
+		</constructor-arg>
+		<property name="propertyLocations">
```

Comment:
@anechifor This breaks when spring tries to create the bean, since there is no setter for propertyLocations property on XMLConfigService; looks like the setter is confusingly named setProperties. I persoanlly would rename the setter to setPropertyLocations within the XMLConfigService, but since might break other uses of that class, we should use properties instead of propertyLocations.